bang/templates/include/costumeTopIllustration.html

```diff
@@ -0,0 +1,173 @@
+{% load tools %}
+{% load i18n %}
+
+{% with costume=item %}
+<p id="noWebGLMessage" class="alert alert-danger" style="text-align: left; display:none;">Sorry, a browser that supports WebGL is required to view Live2D models.</p>
+<p id="loadingMessage" class="text-center fontx2 padding20"><i class="flaticon-loading"></i></p>
+<div id="dlcontainer">
+    <canvas id="dltarget" width="800" height="800" style="max-width:100%; max-height:100%;"></canvas>
```

Comment:
Don't need the variables for size anymore?

Reply:
Yep, the viewer will resize itself after loading